### ISSUE TITLE
RuntimeError on download of IFC after custom pset deleted

### ISSUE THREAD

**Issue by Can-Gr** [ThatOpen/engine_components#523]
### Describe the bug 📝

In any version of IFC, when I create a new custom pset and then remove it with **propertiesManager.removePset(model, expressID)** (which is deleted successfully in the model) and then download the IFC file, I receive error of :

**Download failed: RuntimeError: memory access out of bounds
at web-ifc.wasm:0xcb64c
at web-ifc.wasm:0xca2a2
at Object.RemoveLine (eval at newFunc (web-ifc-api.js:6010:1), :9:1)
at IfcAPI2.DeleteLine (web-ifc-api.js:72239:1)
at me.saveToIfc (index.mjs:11363:1)
at async downloadFile**

But when I delete any existing pset, which comes with the model, and then download the IFC file, there's no error.

I've tested it with different versions of IFCs and sizes.

### System Info 💻

```shell
components: "^2.3.14",
components-front: "^2.3.9",
ui: "^2.3.0",
ui-obc: "^2.3.2",
```

### Used Package Manager 📦

npm

### Validations ✅

- [X] Read the [docs](https://docs.thatopen.com/intro).
- [X] Check that there isn't [already an issue](https://github.com/ThatOpen/engine_components/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a repository issue and not a framework-specific issue. For example, if it's a THREE.js related bug, it should likely be reported to [mrdoob/threejs](https://github.com/mrdoob/three.js) instead.
- [X] Check that this is a concrete bug. For Q&A join our [Community](https://people.thatopen.com/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by HoyosJuan** [comment]
Hey @Can-Gr! 

This has been already solved in [this commit](https://github.com/ThatOpen/engine_components/commit/eb9e0cd1e15a9be8b00de90e085e22038ee489dd) and available in version @thatopen/components@2.3.16. Please, keep in mind while the pset can be deleted now, its IfcRelationship remains in the downloaded file as for now. We are working in some ways to delete relations as noted in issue https://github.com/ThatOpen/engine_components/issues/502.

Let me know if everything is ok to close this issue!

**Comment by Can-Gr** [comment]
Hey @HoyosJuan! Yes, I can delete psets, download the model now and psets are not visible anymore on the element. 
Thank you! 

Although I'm not so sure what would be effect of remaining IfcRelationship, since I don't have a lot of knowledge about IFC structure..
